src/App.vue

```diff
@@ -519,25 +519,38 @@ export default {
 		 * @param {boolean} event.cancelAction Option to cancel the action so no page reload is happening
 		 */
 		interceptNotificationActions(event) {
-			if (event.notification.app !== 'spreed' || event.action.type !== 'WEB') {
+			if (event.notification.app !== 'spreed') {
 				return
 			}
 
-			const [, load] = event.action.url.split('/call/')
-			if (!load) {
-				return
-			}
+			switch (event.action.type) {
+			case 'WEB': {
+				const load = event.action.url.split('/call/').pop()
+				if (!load) {
+					return
+				}
 
-			const [token, hash] = load.split('#')
-			this.$router.push({
-				name: 'conversation',
-				hash: hash ? `#${hash}` : '',
-				params: {
-					token,
-				},
-			})
+				const [token, hash] = load.split('#')
+				this.$router.push({
+					name: 'conversation',
+					hash: hash ? `#${hash}` : '',
+					params: {
+						token,
+					},
+				})
+
+				event.cancelAction = true
+				break
+			}
+			case 'POST': {
```

Comment:
unrelated?

Reply:
Supposed to be there for handling notification actions like "Accept invitation", to redirect a user to federated conversation, but API doesn't return a token for that (as we discussed it earlier)

Reply:
but this PR is about the admin setting, not about notifications? :D

Reply:
So are you just preparing the handling before the API is ready? you can just stash it or will you wait until API PR is created and merged?

Reply:
exactly what Dorra said. The second commit should be a separate PR :)

Reply:
Fiine